packages/in_app_purchase/in_app_purchase_platform_interface/lib/src/in_app_purchase_platform.dart

```diff
@@ -194,4 +194,21 @@ abstract class InAppPurchasePlatform extends PlatformInterface {
   ///    [PurchaseDetails.verificationData].
   Future<void> restorePurchases({String? applicationUserName}) =>
       throw UnimplementedError('restorePurchases() has not been implemented.');
+
+  /// Returns the user's country.
+  ///
+  /// Android:
+  /// Returns Play billing country code based on ISO-3166-1 alpha2 format.
+  ///
+  /// See: https://developer.android.com/reference/com/android/billingclient/api/BillingConfig
+  /// See: https://unicode.org/cldr/charts/latest/supplemental/territory_containment_un_m_49.html
+  ///
+  /// iOS:
+  /// Returns the country code from SKStoreFrontWrapper.
+  ///
+  /// See: https://developer.apple.com/documentation/storekit/skstorefront?language=objc
+  ///
+  ///
+  Future<void> countryCode() =>
+      throw UnimplementedError('countryCode() has not been implemented.');
```

Comment:
You'll need a test to land the stand-alone PR that gets split out of this one. Generally we just test that the default throws Unimplemented.

Reply:
Thanks, added.